index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="header">
+        <h3>Welcome</h3>
+        <a href=""><img src="images/PNG/Enabled.png" alt="Hamburger-icon"></a>
+    </nav>
```

Comment:
- Kindly replace the `nav` tag with a `header` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. :nerd_face: :nerd_face: 
- Please add an `a` tag to your logo. This is for three reasons: convention, semantics, and user experience(UX).  :pray: :pray: 
- Please add a `tag` to your hamburger button. This also is for semantics reasons. :+1: :smiley: :smiley: